### PR TITLE
Implemented layout for ShopX website

### PR DESCRIPTION
**Pull request by v-zinchenko**

### CODE REVIEW

File: safor/debug.log [hunk 1]
```diff
@@ -0,0 +1 @@
+[0626/122123.047:ERROR:crash_report_database_win.cc(428)] unexpected header
```
Comment (volo-maslow) [inline safor/debug.log]: Please remove this useless file.

File: shopx/index.html [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+    <hr>
+    <section class="slider">
+        <div id="demo" class="carousel slide" data-ride="carousel">
+            <ul class="carousel-indicators">
+                <li data-target="#demo" data-slide-to="0" class="active">01</li>
+                <li data-target="#demo" data-slide-to="1">02</li>
+                <li data-target="#demo" data-slide-to="2">03</li>
+            </ul>    
+            <div class="carousel-inner">
+                <div class="carousel-item active" style="background-color: #ade1c5;">
```
Comment (volo-maslow) [inline shopx/index.html]: Why do we need inline styles here? Can it be done with css/sass class?

File: shopx/index.html [hunk 3]
```diff
@@ -0,0 +1,352 @@
...
+                        <li>Latest Jobs</li>
+                        <li>Terms & Conditions</li>
+                        <li>Affiliates</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="col-12 col-lg-4 instagram">
+                <h1><img src="images/footer/instagram.svg" alt=""> Instagram</h1>
+                <ul>
+                    <li><img src="" alt=""></li>
```
Comment (volo-maslow) [inline shopx/index.html]: Should there be some images?

File: shopx/style.css [hunk 4]
```diff
@@ -0,0 +1,1008 @@
+* {
```
Comment (volo-maslow) [inline shopx/style.css:1]: No SASS?

**Review by volo-maslow — CHANGES_REQUESTED** [review]

File: shopx/index.html [hunk 5]
```diff
@@ -0,0 +1,352 @@
...
+                        <li>Latest Jobs</li>
+                        <li>Terms & Conditions</li>
+                        <li>Affiliates</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="col-12 col-lg-4 instagram">
+                <h1><img src="images/footer/instagram.svg" alt=""> Instagram</h1>
+                <ul>
+                    <li><img src="images/footer/instagramm image.jpeg" alt=""></li>
```
Comment (volo-maslow) [inline shopx/index.html]: Please do not use spaces in file names.

**Review by volo-maslow — CHANGES_REQUESTED** [review]

File: shopx/style.scss [hunk 6]
```diff
@@ -0,0 +1,844 @@
+@function calculatePers($pixels, $parrentWidth) {
```
Comment (volo-maslow) [inline shopx/style.scss]: Please remove this function.

**Review by volo-maslow — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Pull request closed without merging**